src/Auth0.AuthenticationApi.Client/Builders/SamlUrlBuilder.cs

```diff
@@ -0,0 +1,20 @@
+namespace Auth0.AuthenticationApi.Client.Builders
+{
+    public class SamlUrlBuilder : UrlBuilderBase<SamlUrlBuilder>
+    {
+        private string ConnectionName { get; set; }
+
+        public SamlUrlBuilder(string baseUrl, string client)
```

Comment:
I would add an additional method here `WithRelayState` or `WithState` that sets the `RleayState` to a value (this is the state).

Ideally this should be a string and an overload with a key value pair (that is then flattened to a string, like `xcrf=abc&ru=/foo` (make sure you also add url encoding here)